be/src/Project/Services/DownloadCenter/DownloadCenter.Api/v1/Controllers/DownloadCenterController.cs

```diff
@@ -0,0 +1,359 @@
+
+            var serviceModel = new GetDownloadCenterFilesCategoryServiceModel
+            {
+                Id = id,
+                Language = CultureInfo.CurrentCulture.Name,
+                Username = this.User.Claims.FirstOrDefault(x => x.Type == ClaimTypes.Email)?.Value,
+                OrganisationId = GuidHelper.ParseNullable(sellerClaim?.Value)
+            };
+
+            var validator = new GetDownloadCenterCategoryFilesModelValidator();
+            var validationResult = await validator.ValidateAsync(serviceModel);
+
+            if (validationResult.IsValid)
+            {
+                var downloadCenterCategory = await this.downloadCenterService.GetDownloadCenterCategoryAsync(serviceModel);
+
+                if (downloadCenterCategory is not null)
+                {
+                    var response = new DownloadCenterCategoryResponseModel
+                    {
+                        Id = downloadCenterCategory.Id,
+                        ParentCategoryId = downloadCenterCategory.ParentCategoryId,
+                        ParentCategoryName = downloadCenterCategory.ParentCategoryName,
+                        CategoryName = downloadCenterCategory.CategoryName,
+                        Subcategories = downloadCenterCategory.Subcategories.OrEmptyIfNull().Select(x => new DownloadCenterSubcategoryResponseModel
+                        {
+                            Id = x.Id,
+                            Name = x.Name
+                        }),
+                        Files = downloadCenterCategory.Files,
+                        LastModifiedDate = downloadCenterCategory.LastModifiedDate,
+                        CreatedDate = downloadCenterCategory.CreatedDate
+                    };
+
+                    return this.StatusCode((int)HttpStatusCode.OK, response);
+                }
+            }
+
+            throw new CustomException(string.Join(ErrorConstants.ErrorMessagesSeparator, validationResult.Errors.Select(x => x.ErrorMessage)), (int)HttpStatusCode.UnprocessableEntity);
+        }
+
+        /// <summary>
+        /// Get list of download center files.
+        /// </summary>
+        /// <param name="searchTerm">The search term.</param>
+        /// <param name="pageIndex">The page index.</param>
+        /// <param name="itemsPerPage">The items per page.</param>
+        /// <param name="orderBy">The optional order by.</param>
+        /// <returns>The list of download center files.</returns>
+        [HttpGet("categories"), MapToApiVersion("1.0")]
+        [ProducesResponseType((int)HttpStatusCode.OK, Type = typeof(PagedResults<IEnumerable<DownloadCenterItemCategoriesResponseModel>>))]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
+        [ProducesResponseType((int)HttpStatusCode.UnprocessableEntity)]
+        public async Task<IActionResult> GetCategories(string searchTerm, int pageIndex, int itemsPerPage, string orderBy)
```

Comment:
**suggestion:** let's make name of this method more meaningful. It hasn't much to do with categories but download center items.